### ISSUE TITLE
Improve the a11y of Avatar component

### ISSUE THREAD

**Issue by julianguyen** [ifmeorg/ifme#2038]
# Description

From the [axe-core](https://github.com/dequelabs/axe-core), Deque's a11y linter:

```
Issue Description
Ensures <img> elements have alternate text or a role of none or presentation

Element location
.HeaderProfile__headerProfile___im69K > .avatar.Avatar__avatar___3vcX_.Avatar__small___19F8v > .lazyload-wrapper > img
Element source
<img class="Avatar__image___2cXeU" src="https://res.cloudinary.com/hy1mfoifn/image/upload/v1605117738/oingt6q3uocvwek8wcib.jpg">
To solve this issue, you need to...
Fix at least one (1) of these issues:
Element does not have an alt attribute

aria-label attribute does not exist or is empty

aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty

Element has no title attribute

Element's default semantics were not overridden with role="none" or role="presentation"
```

Let's update this component to add `alt=""` when no prop value for `alt` is passed into the component!

---

Please assign yourself (via the Assignees dropdown), if you do want to work on this issue. Can't find yourself? You need to [join our organization](https://github.com/ifmeorg/ifme/wiki/Join-Our-Slack).

Check out our [Picking Up Issues](https://github.com/ifmeorg/ifme/wiki/Picking-Up-Issues) guide if you haven't already!
